README.md

```diff
@@ -128,6 +128,29 @@ if (!supportsTime) {
 
 This will add a global `TimePolyfill` function to the page.
 
+## Shadow DOM
+
+When your code is inside a component that resides in the Shadow DOM, the polyfill will not be able to find your label
+element. For this case, you can pass your label element in directly.
+
+```
+<label id="myLabel"></label>
```

Comment:
This is an inaccessible label.

Add a `for="timeInput"` attribute to the label to make it accessible.